examples/kustomization/tenant-letsencrypt/ingress.yaml

```diff
@@ -0,0 +1,34 @@
+apiVersion: networking.k8s.io/v1
+kind: Ingress
+metadata:
+  name: tenant-ingress
+  annotations:
+    kubernetes.io/ingress.class: nginx
```

Comment:
you should add
```yaml
nginx.ingress.kubernetes.io/proxy-body-size: 5t
```

to avoid people opening tickets on why they can't download large files, by default nginx ingress sets this value to `1m`

Reply:
done